internal/output/output.go

```diff
@@ -24,19 +24,24 @@ type Options struct {
 	TemplateData   [][]byte
 }
 
+func (o *Options) isProvidesTemplateData() bool { return o != nil && len(o.TemplateData) > 0 }
```

Comment:
(nit) s/isProvides/provides/g here and throughout. For example ` func (o *Options) isProvidesTemplateData() bool` -> `func (o *Options) providesTemplateData() bool`.

Reply:
I suck at naming things.